### ISSUE TITLE
Crashing of the code while using  gqrs_detect in Cloud Functions

### ISSUE THREAD

**Issue by MishalJasmine** [MIT-LCP/wfdb-python#206]
Hi,

My application requires the QRS calculation to happen on the `Cloud`. So I download the database records from the `Cloud Storage` by setting `pd_dir` in rerecord to  `mitdb`. The problem arises when I use gqrs_detect. 

      for i in range(len(recordlist)):
        record = wfdb.rdrecord(recordlist[i], channels=[0],pb_dir='mitdb')
        test_qrs = processing.gqrs_detect(record.p_signal[:,0], fs=record.fs) // keeps crashing here without any error.

> Function execution took 46397 ms, finished with status: 'crash'

Can I get some help on this?

**Comment by MishalJasmine** [comment]
I assigned the signal from the record to a variable and the Frequency of the record to a variable, both had correct values.

       sig  = record.p_signal[:,0]
       fs  = record.fs

I am not able to figure out where exactly the issue lies.

**Comment by Lucas-Mc** [comment]
Hey @MishalJasmine , I just ran this locally and didn't run into any errors though I do have some comments. To get the `recordlist`, did you run `recordlist = wfdb.get_record_list('mitdb')`? Either way, your resulting `recordlist` should be some kind of subset of the following or else it will fail: 

`['100', '101', '102', '103', '104', '105', '106', '107', '108', '109', '111', '112', '113', '114', '115', '116', '117', '118', '119', '121', '122', '123', '124', '200', '201', '202', '203', '205', '207', '208', '209', '210', '212', '213', '214', '215', '217', '219', '220', '221', '222', '223', '228', '230', '231', '232', '233', '234']`

Also, I ran these commands on the most recent version of WFDB which now extracts all the data from Physionet instead of Physiobank since Physionet is more updated and will be the platform that is updated while Physiobank is slowly phased out. Maybe that is the error? 

One comment though if you update to the most recent version, the command to get the record would now be `record = wfdb.rdrecord(recordlist[i], channels=[0],pn_dir='mitdb')` where it should be noted that the variable `pb_dir` is now called `pn_dir` to reflect the update to Physionet from Physiobank. Let me know your comments and thanks for the inquiry!
